packages/server/package.json

```diff
@@ -56,13 +56,13 @@
     "jsdom": "^15.1.1",
     "json-loader": "^0.5.7",
     "lint-staged": "^10.1.7",
-    "mocha": "^5.2.0",
+    "mocha": "^9.0.3",
     "mockdate": "^2.0.1",
     "prettier": "^1.19.1",
-    "relay-compiler": "^8.0.0",
-    "relay-compiler-language-typescript": "^13.0.2",
-    "relay-config": "^8.0.0",
-    "relay-runtime": "https://github.com/mattkrick/relay/tarball/ea411fa5aa507b9c1082f12e6111bdea7186c817",
+    "relay-compiler": "^11.0.2",
+    "relay-compiler-language-typescript": "^14.0.0",
+    "relay-config": "^11.0.2",
+    "relay-runtime": "^11.0.2",
```

Comment:
this will break our subscription caching & possible a few other things, we'll probably have to fork again to upgrade

Reply:
Is this a simple fix, if you could point me at what needs to change, I may be able to help?

Reply:
@mattkrick I reverted per your guidance, thank you sir!